tools/cli/helpers/list-project-files.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import readline from 'readline';
+
+/**
+ * Internal dependencies
+ */
+import FilterStream from './filter-stream.js';
```

Comment:
In addition to removing the need for the comments, a linting rule now also wants no blank lines in the list.
```suggestion
import readline from 'readline';
import FilterStream from './filter-stream.js';
```